### ISSUE TITLE
Unable to locate package dovecot-lua

### ISSUE THREAD

**Issue by schildbach** [dovecot/docker#7]
In the 2.3.21 directory, `docker build .` leads to:

```
#8 [4/5] RUN apt-get -y update && apt-get -y install   tini   dovecot-core   dovecot-gssapi   dovecot-imapd   dovecot-ldap   dovecot-lmtpd   dovecot-lua   dovecot-managesieved   dovecot-mysql   dovecot-pgsql   dovecot-pop3d   dovecot-sieve   dovecot-solr   dovecot-sqlite   dovecot-submissiond   ca-certificates   ssl-cert &&   rm -rf /var/lib/apt/lists &&   groupadd -g 1000 vmail &&   useradd -u 1000 -g 1000 vmail -d /srv/vmail &&   passwd -l vmail &&   rm -rf /etc/dovecot &&   mkdir /srv/mail &&   chown vmail:vmail /srv/mail &&   make-ssl-cert generate-default-snakeoil &&   mkdir /etc/dovecot &&   ln -s /etc/ssl/certs/ssl-cert-snakeoil.pem /etc/dovecot/cert.pem &&   ln -s /etc/ssl/private/ssl-cert-snakeoil.key /etc/dovecot/key.pem
#8 sha256:6c8895a87a454e0ec47d5150bdf3f29601c617b00ae4029285348417cb6e8819
#8 0.636 Err:1 https://repo.dovecot.org/ce-2.3.21/debian/bullseye bullseye InRelease
#8 0.636   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 94.237.82.12 443]
#8 0.677 Get:2 http://deb.debian.org/debian bullseye InRelease [116 kB]
#8 0.745 Get:3 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
#8 0.769 Get:4 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
#8 0.894 Get:5 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
#8 1.619 Get:6 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [246 kB]
#8 1.640 Get:7 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [17.5 kB]
#8 2.226 Fetched 8655 kB in 2s (5235 kB/s)
#8 2.226 Reading package lists...
#8 2.492 W: https://repo.dovecot.org/ce-2.3.21/debian/bullseye/dists/bullseye/InRelease: No system certificates available. Try installing ca-certificates.
#8 2.492 W: Failed to fetch https://repo.dovecot.org/ce-2.3.21/debian/bullseye/dists/bullseye/InRelease  Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 94.237.82.12 443]
#8 2.492 W: Some index files failed to download. They have been ignored, or old ones used instead.
#8 2.498 Reading package lists...
#8 2.764 Building dependency tree...
#8 2.826 Reading state information...
#8 2.877 E: Unable to locate package dovecot-lua
```

**Comment by cmouse** [comment]
I'll check. It should have worked because I built the image last week using this?

**Comment by cmouse** [comment]
Fixed with e2e524789ada866d8b63ea2dba0a9bf7e0f0cc77

**Comment by schildbach** [comment]
> Fixed with e2e524789ada866d8b63ea2dba0a9bf7e0f0cc77

That commit is not yet pushed.

**Comment by cmouse** [comment]
Yeah, sorry. It's now there.
